geomstats/visualization.py

```diff
@@ -20,7 +21,7 @@
 
 AX_SCALE = 1.2
 
-IMPLEMENTED = ['SO3_GROUP', 'SE3_GROUP', 'S1', 'S2',
+IMPLEMENTED = ['SO3_GROUP', 'SE3_GROUP', 'SE2_GROUP', 'S1', 'S2',
```

Comment:
(this `IMPLEMENTED` list is not very clean, it could probably be replaced by something else.)